addons/udes_stock_refactoring/models/registry.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from abc import ABCMeta
+from addons.udes_common.models.models import groupby
+
+
+class RegistryMeta(ABCMeta):
```

Comment:
As suggested locations and this is using this meta do we want to move that into the base module with the groupby and the other tools

Reply:
Have now moved this into `udes_common` and updated `udes_stock_refactoring` and `suggested_locations` to take the RegistryMeta from there.